### PR TITLE
RaR: Fast Break

### PR DESCRIPTION
**Pull request by NoahTheDuke**
Implemented with a fairly robust test. Adds install reqs to the "Root only" upgrades, as our process for handling them is hacky and UI reliant. This at least future-proofs us while not affecting current functionality.

### CODE REVIEW

**Comment by Saintis** [comment]
Maybe go the full distance and get rid of the `:init` stuff on the root-only upgrades. The new system is much better anyway!

**Comment by NoahTheDuke** [comment]
I don't want to change that stuff until we switch over to the Regeant branch, cuz all cljs work we do adds work to poor @danhut. I made a note to myself about this, I'll definitely revisit it once we switch over.

File: src/clj/game/cards/operations.clj [hunk 1]
```diff
@@ -478,6 +478,43 @@
...
+                               {:prompt "Select a card to install"
+                                :choices {:req #(and (= (:side %) "Corp")
+                                                     (not (is-type? % "Operation"))
+                                                     (in-hand? %)
+                                                     (seq (filter (fn [c] (= server c)) (corp-install-list state %))))}
+                                :effect (req (wait-for
+                                               (corp-install state side target server nil)
+                                               (let [server (if (= "New remote" server)
+                                                              (-> (turn-events state side :corp-install)
+                                                                  ffirst :zone second zone->name)
```
Comment (nealterrell) [inline src/clj/game/cards/operations.clj]: I'm having CADDAADAR flashbacks.

Comment (NoahTheDuke) [reply src/clj/game/cards/operations.clj]: Hahaha I had the same feeling.

**Pull request merged**